src/api/controllers/db-controller.ts

```diff
@@ -86,13 +87,22 @@ function getAssetEventTypeString(
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Block extends DbBlock {}
```

Comment:
```suggestion
export type Block = DbBlock;
```

should remove need for eslint override?